generator/test/error_test_data/asymmetric_external_link.expected

```diff
@@ -0,0 +1,12 @@
+Linking to asymmetric objects is not allowed
+
+in: asset:pkg/test/error_test_data/asymmetric_external_link.dart:12:16
+   ╷
+10 │ @RealmModel()
+11 │ class _Bad {
+   │       ━━━━ in realm model for 'Bad'
+12 │   _Asymmetric? asymmetric;
+   │                ^^^^^^^^^^ !
+   ╵
+Remove the field
```

Comment:
It would be nice to give to the user some more explanation about this error "Remove the field".

Reply:
The header line of the error is: _Linking to asymmetric objects is not allowed_

Isn't that the context you need? The last line is the suggested action. The only other option would be to make `_Asymmetric` not be asymmetric. Should we suggest that as well?

Reply:
Sorry, I've missed the title. Yes, it is enough.